mmstudio/src/main/java/org/micromanager/internal/hcwizard/PropertyTableModel.java

```diff
@@ -50,7 +51,9 @@ class PropertyTableModel extends AbstractTableModel implements MMPropertyTableMo
    PropertyItem[] props_;
    String[] devNames_;
    DeviceSetupDlg setupDlg_;
-   
+   CMMCore core_;
+
+   // OK to delete?

```

Comment:
If it's not used, it can be deleted (and probably should be).

It also looks like `mode` is never other than `PREINIT`, meaning that `updateValues` could be simplified, but I digress.